koin-projects/koin-androidx-viewmodel/src/main/java/org/koin/androidx/viewmodel/ext/android/ActivityExt.kt

```diff
@@ -29,35 +31,42 @@ import kotlin.reflect.KClass
  *
  * @author Arnaud Giuliani
  */
-
-fun <T : ViewModel> ViewModelStoreOwner.viewModel(
-    clazz: KClass<T>,
+inline fun <reified T : ViewModel> AppCompatActivity.viewModel(
```

Comment:
Better to use [ComponentActivity](https://developer.android.com/reference/androidx/activity/ComponentActivity)